test/data/default.js

```diff
@@ -78,19 +78,19 @@ module.exports = models => {
         },
         {
           id: 'a942ffc7-e7ca-4d76-a001-0b5048a057d1',
-          firstName: 'Li Sen',
-          lastName: 'Xing',
-          email: 'lisenxing@example.com',
-          asruUser: true,
-          asruLicensing: true
-        },
-        {
-          id: 'a942ffc7-e7ca-4d76-a001-0b5048a057d2',
           firstName: 'Inspector',
           lastName: 'Morse',
           email: 'inspector-morse@example.com',
           asruUser: true,
           asruInspector: true
+        },
+        {
+          id: 'a942ffc7-e7ca-4d76-a001-0b5048a057d2',
+          firstName: 'Li Sen',
+          lastName: 'Xing',
+          email: 'lisenxing@example.com',
+          asruUser: true,
+          asruLicensing: true
```

Comment:
The id for inspectors and licensing were transposed in the seeds when compared with https://github.com/UKHomeOffice/asl-workflow/blob/master/test/data/profiles.js

Reply:
Same for the tests - could've just fixed them in profiles.js but I'd already done half of them by that point